src/chart/Line.js

```diff
@@ -22,18 +22,32 @@
 
     Line.prototype.updateChart = function (domNode, element, margin, width, height) {
         var context = this;
-
+        var d3Line;
         this._palette = this._palette.switch(this.paletteID());
-        var d3Line = d3.svg.line()
-            .x(function (d) {
-                switch (context._xScale) {
-                    case "DATE":
-                        return context.x(context.parseDate(d[0]));
-                }
-                return context.x(d[0]) + (context.x.rangeBand ? context.x.rangeBand() / 2 : 0);
-            })
-            .y(function (d) { return context.y(d[1]); })
-        ;
+
+        if (this.orientation() === "horizontal") {
+            d3Line = d3.svg.line()
+                .x(function (d) {
+                    switch (context._xScale) {
+                        case "DATE":
+                            return context.x(context.parseDate(d[0]));
+                    }
+                    return context.x(d[0]) + (context.x.rangeBand ? context.x.rangeBand() / 2 : 0);
+                })
+                .y(function (d) { return context.y(d[1]); })
+            ;
+        } else {
+            d3Line = d3.svg.line()
+                .y(function (d) {
+                    switch (context._xScale) {
+                        case "DATE":
+                            return context.x(context.parseDate(d[0]));
+                    }
+                    return context.x(d[0]) + (context.x.rangeBand ? context.x.rangeBand() / 2 : 0);
+                })
+                .x(function (d) { return context.y(d[1]); })
+            ;
+        }
```

Comment:
Above the two functions used in the .x and .y calls don't change so you can declare them in one place (beside the ```var d3Line```) and just refer to them inside the if/else block?